depends/packages/rapidcheck.mk

```diff
@@ -0,0 +1,13 @@
+package=rapidcheck
+$(package)_version=f5d3afa
+$(package)_download_path=https://bitcoin-10596.firebaseapp.com/depends
```

Comment:
We'll probably want to put this on `https://bitcoincore.org/depends-sources`

Reply:
How do I go about this? Is there a process document some where?

Reply:
This needs to be done by someone who has push access to that folder. Can be done after (or just before) this pull is merged.